File: src/basic_training/Day09_3.java
package basic_training;

public class Day09_3 {
	public String solution(String my_string, int n) {
		return my_string.substring(my_string.length() - n);
	}
}

// 자바 문자열 자르기
// 문자열.substring();
//사용법
// String.substring(start) //문자열  start위치부터 끝까지 문자열 자르기
// String.substring(start,end) //문자열  start위치 부터 end전까지 문자열 발췌		
// 1. 마지막 3글자 자르기
// String str = "ABCDEFG"; 
// String result = str.substring(str.length()-3, str.length());
// System.out.println(result)
// 결과값EFG
// 참조: https://byul91oh.tistory.com/309

File: src/basic_training/Day09_3.js
function solution(my_string, n) {
	return my_string.slice(my_string.length - n);
}

// 자바스크립트 문자열 길이
// 문자열.length

// 자바스크립트 문자열 자르기
// 문자열.slice
// substring 과 비슷하나, 
// 차이점은 인자가 음수일때, substring 은 빈배열 반환 / slice 는 문자열 반환(뒤에서부터 인덱스 적용)

// 문자열.substring(start, end) : 시작인덱스에서 끝 인덱스전까지 문자열 반환

// slit - 구분자로 자르기 => 배열 반환
// const str = "hello, world, javascript";
// str.split(',');
// str = ['hello', 'world', 'javascript']